src/style.css

```diff
@@ -1,3 +1,102 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Arial, sans-serif;
+  background-color: grey;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  line-height: 60px;
+  height: 60px;
+}
+
+@media (min-width: 1200px) {
+  .nav {
+    margin-left: 419px;
+  }
+}
+
+@media (min-width: 1024px) {
+  .nav {
+    margin-left: 243px;
+  }
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  text-transform: uppercase;
+  height: 60px;
+}
+
+.img {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  padding-left: 50px;
+}
+
+.box {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-align: left;
+  text-underline-position: from-font;
+  text-decoration: none;
+  font-weight: 500;
+  color: #000;
+  margin-right: 20px;
+}
+
+.box-8 {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-align: left;
+  text-underline-position: from-font;
+  text-decoration: none;
+  font-weight: 500;
+  color: #000;
+  margin-right: 50px;
+}
```

Comment:
Redundant class definition: The styles for '.box-8' are almost identical to those for '.box'. Consider removing '.box-8' and applying the '.box' class to the corresponding HTML element for consistency and to avoid redundancy.